### PR TITLE
Fix/37716 textdomain change

### PR DESCRIPTION
**Pull request by barryhughes**
Per [C#37716](https://central.tri.be/issues/37716#note-5) this PR updates the text domain to match the plugin slug.

### CODE REVIEW

File: the-events-calendar.php [hunk 1]
```diff
@@ -2,7 +2,7 @@
 /*
 Plugin Name: The Events Calendar
 Description: The Events Calendar is a carefully crafted, extensible plugin that lets you easily share your events. Beautiful. Solid. Awesome.
-Version: 3.12
+Version: 3.12.1dev
```
Comment (borkweb) [inline the-events-calendar.php]: Should this be `3.12.1` rather than `3.12.1dev`?

**Comment by borkweb** [comment]
This was incredibly boring and easy to review. Which is a testament to the success of the changeset.

**Comment by borkweb** [comment]
:+1:

File: the-events-calendar.php (hunk 1 above)
Comment (barryhughes) [inline the-events-calendar.php]: Dropped the dev suffix

**Pull request merged**